### PR TITLE
DOCS-9304 December release

### PR DESCRIPTION
**Pull request by wwarner-salesforce**
https://docs.google.com/document/d/1K7RTS6hdKpG4oYsXGFO6_tYZw79XClg12LveRgHHsnk/

Install guide: https://docs.google.com/document/d/13QvlRtJmcMWWlFyyv1Knsn7_REbwc6tEiB1nqugceDQ/

### CODE REVIEW

**Review by ferferrarazzo — DISMISSED** [review]

File: modules/ROOT/pages/diagnostics.adoc [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+. *Package version* shows the version of your currently installed API Community Manager package.
+. *Connectivity with Anypoint* shows the URL your API community uses to access Anypoint.
+. *Cache*:
+.. *Org cache enabled* shows if the organization cache is enabled and links to the platform cache metadata.
+.. *Org cache allocated* shows the size of the organization cache allocated and links to the platform cache configuration.
+.. *Session cache enabled* shows if the session cache is enabled and links to the platform cache metadata.
+.. *Session cache allocated* shows the size of the session cache allocated and links to the platform cache configuration.
+
+The *ACM Communities* section shows each of your ACM communities.
+
```
Comment (ferferrarazzo) [inline modules/ROOT/pages/diagnostics.adoc:20]:
```suggestion
@wwarner-salesforce  At this point the user will not have any community and we would have to create the link to return to create community and continue with the configuration.
We would also have to document in the event that the client does have communities.

Something like this --->
== Next Steps

After the connection with your Anypoint organization is established, and the diagnoses were reviewed , click *Create Community* to open the Create Community page xref:create-community.adoc[Create a Community] to create your first community.
```

Comment (ferferrarazzo) [reply modules/ROOT/pages/diagnostics.adoc:20]: @electronwill @jdeoliveira

File: modules/ROOT/pages/update-acm.adoc [hunk 2]
```diff
@@ -21,6 +21,9 @@ The staging environment will have a partial or full copy of the data in the prod
 
 To update your API community:
 
+. Before performing any steps, read all the steps, including pre-install and post-install.
++
+Some steps may require mandatory downtime during the update.
```
Comment (ferferrarazzo) [inline modules/ROOT/pages/update-acm.adoc]: @wwarner-salesforce we should change this, Jesus told me to put something like this: 

potential degraded service of your community: APIs, docs and client apps pages may not work after the upgrade until you perform the required config changes. Notify your customers and stakeholders as you see fit.

File: modules/ROOT/pages/create-community.adoc [hunk 3]
```diff
@@ -219,6 +219,10 @@ These are the permissions for the member users:
 |*Read*
 |*Read Access*
 
+|CommunityAssetVersions
```
Comment (ferferrarazzo) [inline modules/ROOT/pages/create-community.adoc:234]: @wwarner-salesforce here we could add some * or a clarification to notify the user that some fields were introduced in the last version of the docs and point them to the release notes or the upgrade page.
- if you are in an older version you wont' see all objects. in your org"

File: modules/ROOT/pages/data-bridge.adoc [hunk 4]
```diff
@@ -1,4 +1,4 @@
-= Set Up the Data Bridge
+= Manual Setup: Set Up the Data Bridge
```
Comment (ferferrarazzo) [inline modules/ROOT/pages/data-bridge.adoc]: We could move this to the Manual setup section

Comment (ferferrarazzo) [reply modules/ROOT/pages/data-bridge.adoc]: And ww should move the Prepare Identity Provider Data to the prerequisites and use a pointer here.

**Review by ferferrarazzo — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/diagnostics.adoc (hunk 1 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/diagnostics.adoc:20]: Thank you! Fixed.

File: modules/ROOT/pages/update-acm.adoc (hunk 2 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/update-acm.adoc]: Updated: "If necessary, notify your customers and stakeholders of the update, and that APIs, docs, and client app pages may not work during the update during the time after installing the new package and before completing the required configuration changes."

File: modules/ROOT/pages/create-community.adoc (hunk 3 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/create-community.adoc:234]: Updated: "These permissions reflect the current API Community Manager package version. If you are using an older version, your member user profile may have only some of these permissions objects. Set the objects in your member user profile to match the table, and ignore objects that are not in your member user profile."

File: modules/ROOT/pages/data-bridge.adoc (hunk 4 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/data-bridge.adoc]: Done.

**Review by ferferrarazzo — DISMISSED** [review]

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 5]
```diff
@@ -327,7 +327,9 @@ Show "Learn More" Button:: Shows the "Learn More" button.
 API Learn More Button Label:: Specifies the button label for APIs. Default is *Learn More*.
 API Group Learn More Button Label:: Specifies the button label for API groups. Default is *Learn More about this API Group*.
 Show API Access Requester:: Shows the API access requester.
-Show API Version:: Shows the asset version from Exchange.
+Show API Version:: Shows the asset version from Exchange when only one version is visible.
+
+When more than one version of the asset is visible, the API Header component displays a version selector and the Show API Version checkbox has no effect.
```
Comment (priyaayyar) [inline modules/ROOT/pages/acm-lightning-components.adoc]: *Show AP Version*

Comment (priyaayyar) [reply modules/ROOT/pages/acm-lightning-components.adoc]: In Bold

File: modules/ROOT/pages/create-community.adoc [hunk 6]
```diff
@@ -369,25 +428,19 @@ Configure sharing settings for guests:
 .. *Field*: `Visibility` *Operator*: `equals` *Value*: `Public`
 . In *Select the users to share with*, select `[Community Name] Site Guest User`.
 . In *Select the level of access for the users*, select *Read Only*.
-. Click Save.
```
Comment (priyaayyar) [inline modules/ROOT/pages/create-community.adoc:372]: In Bold?

File: modules/ROOT/pages/acm-lightning-components.adoc (hunk 5 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/acm-lightning-components.adoc]: Good catch! Fixed.

File: modules/ROOT/pages/create-community.adoc (hunk 6 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/create-community.adoc:372]: Since this is now changed to "Save.", is it correct as it is.

**Review by ferferrarazzo — APPROVED** [review]

**Review by priyaayyar — APPROVED** [review]

**Pull request merged**